Atomic/scan.py

```diff
@@ -517,3 +533,7 @@ def write_persistent_data(self):
 
         with open(summary_file, 'w') as f:
             json.dump(persistent_data, f, indent=4)
+
+
+    def remediate(self, script, iid, results_dir):
+        util.check_call([sys.executable, script, 'id={}'.format(iid), 'results_dir={}'.format(results_dir)])
```

Comment:
@baude Could it be `--id` and `--results_dir`?

Reply:
+100
@baude could this please follow standard UNIX command line conventions? Would enable us to use argparse and get a lot of safeties for free in the script.

Reply:
ok